gulpfile.js

```diff
@@ -1,261 +1,18 @@
 var gulp = require('gulp');
-var gulpif = require('gulp-if');
-var gutil = require('gulp-util');
-var imagemin = require('gulp-imagemin');
-var less = require('gulp-less');
-var csso = require('gulp-csso');
-var mkdirp = require('mkdirp');
-var path = require('path');
-var replace = require('gulp-replace');
-var runSequence = require('run-sequence');
-var sourcemaps = require('gulp-sourcemaps');
-var uglify = require('gulp-uglify');
-var webpack = require('webpack');
-var WebpackNotifierPlugin = require('webpack-notifier');
-
-var config = require('./.build.config');
-var packageInfo = require('./package');
-var webpackConfig = require('./.webpack.config');
-
-var development = process.env.NODE_ENV === 'development';
-var devBuild = development || (process.env.NODE_ENV === 'testing');
-var externalPluginsDirectory = '../dcos-ui-plugins-private';
-
-var pluginsGlob = [
-  externalPluginsDirectory + '/**/*.*'
-];
-
-function browserSyncReload() {
-  if (development) {
-    browserSync.reload();
-  }
-}
-// Watches for delete in external plugins directory and deletes counterpart in
-// DC/OS-UI directory
-function deletePluginFile(event) {
-  if (event.type === 'deleted') {
-    var filePathFromPlugins = path.relative(
-      path.resolve(externalPluginsDirectory), event.path);
-
-    var destFilePath = path.resolve(
-      config.dirs.pluginsTmp, filePathFromPlugins);
-
-    del.sync(destFilePath);
-    webpackFn(browserSyncReload);
-  }
-}
 
-function ensurePluginDirectoryExists() {
-  // Make temp plugins directory if doesn't exist
-  if (!fs.existsSync(config.dirs.pluginsTmp)) {
-    mkdirp(config.dirs.pluginsTmp, function () {
-      console.log('Created ' + config.dirs.pluginsTmp);
-    });
+gulp.task('ensureConfig', function () {
+  // Make sure we have a Config.dev so we don't error on Config loading
+  var configFilePath = './src/js/config/Config.dev.js';
+  if (!fs.existsSync(configFilePath)) {
+    fs.writeFileSync(configFilePath, 'module.exports = {};', 'utf8');
   }
-}
-ensurePluginDirectoryExists();
-
-// Clean out plugins in destination folder
-gulp.task('clean:external-plugins', function () {
-  return del([config.dirs.pluginsTmp + '/**/*']);
-});
-
-// Copy over all
-gulp.task('copy:external-plugins', ['clean:external-plugins'], function () {
-  return gulp.src(pluginsGlob)
-    .pipe(gulp.dest(config.dirs.pluginsTmp));
-});
-
-// Copy over changed files
-gulp.task('copy:changed-external-plugins', function () {
-  return gulp.src(pluginsGlob)
-    .pipe(changed(config.dirs.pluginsTmp))
-    .pipe(gulp.dest(config.dirs.pluginsTmp));
-});
-
-gulp.task('browsersync', function () {
-  browserSync.init({
-    online: true,
-    open: false,
-    port: 4200,
-    server: {
-      baseDir: config.dirs.dist
-    },
-    socket: {
-      domain: 'localhost:4200'
-    }
-  });
 });
 
-gulp.task('connect:server', function () {
+gulp.task('serve', function () {
   connect.server({
     port: 4200,
     root: config.dirs.dist
```

Comment:
Not for me ![](http://cl.ly/0f2U0x022a0B/Image%202016-05-03%20at%2013.26.30.png)